devito/arch/compiler.py

```diff
@@ -771,6 +771,20 @@ def __init_finalize__(self, **kwargs):
             self.__init_intel_mpi__()
             self.__init_intel_mpi_flags__()
 
+        if configuration['profiling'] == 'advisor':
+            # Locate the Intel Advisor installation and
+            # add the necessary flags to the compiler
+            from devito.operator.profiling import locate_intel_advisor
+            path = locate_intel_advisor()
```

Comment:
Why aren't all those set by the `setvars.sh` that's what it's supposed to be for

Reply:
Unfortunately, this was true with Intel studios, but now they are not by default. You have to link explicitly. At least, this is what I see.

Reply:
And this is actually why it stopped working at some point